containers/web-server/Dockerfile-dev

```diff
@@ -0,0 +1,43 @@
+# multistage build of nginx image
+
+# stage 1 of 2
+
+# name the node stage "builder"
+FROM node:16 AS builder
+WORKDIR /app
+# copy package.json and install -- do this before copying everything else
+# to enable more efficient caching by docker (we don't reinstall packages when
+# we've only changed application code)
+#COPY package.json .
+COPY frontend/package.json .
+# install node modules
+RUN npm install
+# copy folders and files required for build into image N.B. when docker copy is given a
+# directory, it copies the contents but not the dir itself, so you must name the destination dir
+#   COPY images images
+#   COPY public public
+#   COPY src src
+#   COPY .eslintrc.js tsconfig.json tsconfig-typings.json .
+COPY frontend .
+# build to static site
+RUN npm run build
```

Comment:
Perhaps this is comment to turn into an issue and put lower down the dev-experience TODO-list.. could we mount the source directory and run `npm start` to pick up frontend changes and live reload? Would need a fatter single-stage container, possibly just with node and react-scripts configured to proxy connections to other services.

Reply:
I added live reload into the frontend last week and its been working well for me here using npm start outside the dev container (accessing backend services hosted within docker and exposed to localhost)